openapi.json

```diff
@@ -0,0 +1,85 @@
+{
+  "openapi": "3.0.3",
+  "info": {
+    "title": "upld description",
+    "license": {
+      "name": "AGPL",
+      "url": "https://www.gnu.org/licenses/agpl-3.0.en.html"
+    },
+    "version": "1.0.11"
+  },
+  "servers": [
+    {
+      "url": "https://upld.risorse.students.cs.unibo.it"
+    }
+  ],
+  "paths": {
+    "/callback": {
+      "post": {
+        "tags": [
+          "auth"
+        ],
+        "summary": "Callback received from the GitHub OAuth2 flow",
+        "parameters": [
+          {
+            "in": "query",
+            "name": "redirect",
+            "description": "The url to be redirected when the auth is successful. In the redirect response, it will also set the cookie and relative expiry date for the upld domain.",
```

Comment:
Metterei il path di dynamic al posto di un open redirect, è una vuln

Reply:
Il fatto è che dynamik potrebbe avere più istanze in futuro. Massimo mettiamo dei controlli client side, o semplicemente blocchiamo le richieste con le CORS.